daemons/fenced/fenced_history.c

```diff
@@ -347,6 +347,21 @@ stonith_merge_in_history_list(GHashTable *history)
 
         updated = TRUE;
         g_hash_table_iter_steal(&iter);
+
+        if ((op->state != st_failed) &&
+            (op->state != st_done) &&
```

Comment:
I think we check the above in enough places that we should really have a convenience function (though not necessary with this PR).

Reply:
Yep, just one more place to add to a subsequent PR that introduces the convenience function ;-)